tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "target": "es2020",
     "module": "es2020",
     "types": ["svelte", "vite/client"],
+    "lib": ["ES2021.String", "DOM"],
```

Comment:
This is needed to let Typescript know of `String.prototype.replaceAll` which has been introduced in ES2021